src/main/java/com/rarchives/ripme/ripper/rippers/InstagramRipper.java

```diff
@@ -145,14 +152,14 @@ private String getMedia(JSONObject data) {
                 for (int carouselIndex = 0; carouselIndex < carouselMedias.length(); carouselIndex++) {
                     JSONObject carouselMedia = (JSONObject) carouselMedias.get(carouselIndex);
                     String imageURL = getMedia(carouselMedia);
-                    if (!imageURL.equals("")) {
+                    if (!"".equals(imageURL)) {
```

Comment:
Yoda conditionals are not necessary when using methods (like `.equals` because there is no risk of using `=` instead of `==`) and besides that, this reads fairly bizarrely. There's no real reason to make this change, as it's not part of the bug fix.